src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs

```diff
@@ -23,7 +23,7 @@ protected override bool IsValid(PropertyValidatorContext context)
                 return true;
             }
 
-            if (value >= 10 && value <= 120)
+            if (value >= 60 && value <= 720)
```

Comment:
Can you change that to 10 again? We still want users without rate limits to go pretty low